content/2-3.md

```diff
@@ -170,49 +175,52 @@ Students should be able to:
 * mark up additional descriptions for images using one of the following techniques:
   * HTML elements `figure` and `figcaption`
   * WAI-ARIA attribute `aria-describedby`
-  * HTML attribute `longdesc` (especially in eBook publications where it is more broadly supported)
-* style text decorations using CSS Transforms and CSS Fonts technologies instead of using images of text
-* code mathematical expressions using the MathML language instead of using images of text
-* categorize images based on the following types:
-  * textual
-  * complex
-* summarize related requirements for authors to write meaningful text alternatives for textual and complex images
+  * HTML attribute `longdesc` (in certain contexts, such as ePub)
+* style text using CSS Transforms and CSS Fonts instead of using images of text
+* code mechanisms that update text alternatives when images are changed dynamically, including animations
+* explain how to provide accessible charts, diagrams, and infographics by using SVG instead of non-scalable image
```

Comment:
I think the last word is plural: "non-scalable images"?

Reply:
Yes, should be. Thanks for pointing it out, will update.